src/api_clients/pocket_publisher_database_execute.py

```diff
@@ -0,0 +1,97 @@
+import os
+from typing import Any
+
+from prefect.tasks.mysql import MySQLExecute
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class PocketPublisherDatabaseExecute(MySQLExecute):
+    """
+    PocketPublisherDatabaseQuery automatically connects to our Publisher MySQL Database if the right environment variables are set:
+    - POCKET_PUBLISHER_DATABASE_HOST
+    - POCKET_PUBLISHER_DATABASE_PORT
+    - POCKET_PUBLISHER_DATABASE_DBNAME
+    - POCKET_PUBLISHER_DATABASE_USER
+    - POCKET_PUBLISHER_DATABASE_PASSWORD
+    """
+
+    def __init__(
+            self,
+            query: str = None,
+            database_host_env_var_name: str = "POCKET_PUBLISHER_DATABASE_HOST",
+            database_port_env_var_name: str = "POCKET_PUBLISHER_DATABASE_PORT",
+            database_dbname_env_var_name: str = "POCKET_PUBLISHER_DATABASE_DBNAME",
+            database_user_var_name: str = "POCKET_PUBLISHER_DATABASE_USER",
+            database_password_var_name: str = "POCKET_PUBLISHER_DATABASE_PASSWORD",
+            cursor_type: str = 'cursor',
```

Comment:
[MySQLExecute](https://docs-v1.prefect.io/api/latest/tasks/mysql.html#mysqlexecute) doesn't have a `cursor_type` argument. Not sure if it would produce any errors, but probably better to remove all references to `cursor_type`.
```suggestion
```